pages/history.tsx

```diff
@@ -9,27 +9,79 @@ import TransactionsContainer from 'sections/history/TransactionsContainer';
 import StatsSection from 'components/StatsSection';
 
 import StatBox from 'components/StatBox';
-import useSynthetixQueries from '@synthetixio/queries';
+import { issuance, SynthetixQueryContext } from '@synthetixio/queries';
 import { useRecoilValue } from 'recoil';
 import { walletAddressState } from 'store/wallet';
+import { StakingTransactionType } from 'sections/history/types';
+import _ from 'lodash';
+import React from 'react';
 
 const HistoryPage: FC = () => {
 	const { t } = useTranslation();
 
 	const walletAddress = useRecoilValue(walletAddressState);
 
-	const { useFeeClaimHistoryQuery } = useSynthetixQueries();
+	const issuanceURL =
+		React.useContext(SynthetixQueryContext)?.context.subgraphEndpoints.issuance || '';
+	const issues = issuance.useGetIssueds(
+		issuanceURL,
+		{
+			first: 1000,
+			orderBy: 'timestamp',
+			orderDirection: 'desc',
+			where: { account: walletAddress?.toLowerCase() },
+		},
+		{ id: true, timestamp: true, value: true }
+	);
+	const burns = issuance.useGetBurneds(
+		issuanceURL,
+		{
+			first: 1000,
+			orderBy: 'timestamp',
+			orderDirection: 'desc',
+			where: { account: walletAddress?.toLowerCase() },
+		},
+		{ id: true, timestamp: true, value: true }
+	);
+	const feeClaims = issuance.useGetFeesClaimeds(
+		issuanceURL,
+		{
+			first: 1000,
+			orderBy: 'timestamp',
+			orderDirection: 'desc',
+			where: { account: walletAddress?.toLowerCase() },
+		},
+		{ id: true, timestamp: true, rewards: true, value: true }
+	);
 
-	const feesClaimedQuery = useFeeClaimHistoryQuery(walletAddress);
 	const { setTitle } = UIContainer.useContainer();
 
-	const isLoaded = feesClaimedQuery.isSuccess;
-	const feesClaimed = feesClaimedQuery.data ?? [];
+	const isLoaded = issues.isSuccess && burns.isSuccess && feeClaims.isSuccess;
+
+	const history = isLoaded
+		? _.sortBy(
+				_.flatten([
```

Comment:
You should be able to just use `flat` ie. `[[]].flat()`

Reply:
https://github.com/Synthetixio/staking/pull/769/commits/0a9be4a938eb698efd5a4b16244d182fdacbbbb5